lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -114,11 +120,15 @@ export default function ErrorDisplay({ children, description = '', error }) {
 
   return (
     <Scrollbox classes="LMS-Scrollbox">
-      <div className="hyp-u-vertical-spacing hyp-u-padding--top--4">
-        {message && <p data-testid="error-message">{toSentence(message)}</p>}
+      <div className="hyp-u-vertical-spacing pt-4 space-y-4">
```

Comment:
One of my tasks, this week I hope, is to deal with the ongoing battle of margins on paragraphs. I expect LMS will end up with a CSS reset, at least for the `frontend_apps` CSS, in short order. Until then, I'm retaining the use of `hyp-u-vertical-spacing`, which will enforce that the first child (which may be a paragraph) has no top margin, and also explicitly disabling margins on the local paragraph elements. Expect to see movement around this quite soon.